Lib/pprint.py

```diff
@@ -342,6 +342,32 @@ def _pprint_mappingproxy(self, object, stream, indent, allowance, context, level
 
     _dispatch[_types.MappingProxyType.__repr__] = _pprint_mappingproxy
 
+    def _pprint_simplenamespace(self, object, stream, indent, allowance, context, level):
+        write = stream.write
```

Comment:
It's a little thing, but there's no real need to do this lookup ahead of time (presumably for the sake of the loop).  It just adds one extra thing to keep track of mentally when reading this method.

The only benefit would be if there were a massive amount (e.g. millions) of items in the namespace (and pprint was a critical part of performance for the code).  However, at that point I doubt you'd be using pprint.

Reply:
Oh yeah, I mostly did this because some of the other methods did. Stupid me didn't even use it on every call to write.

Reply:
This is an optimization used whether ever in this module. The gain is tiny, but it is accumulated, and there is no reason to remove it in this particular function.